content/blog/abstract-classes-and-interfaces/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Abstract Classes and Interfaces
+date: "2019-04-17T21:37:03.284Z"
```

Comment:
Can we backdate some of this content, so that it's listed in a way that makes sense?